### DISCUSSION TITLE
FTB Mobpack setup question

### DISCUSSION THREAD

**Discussion by TheDoibs** [itzg/docker-minecraft-server#1764] [Q&A]
Reading in your readme file you mention that you can just hover over the modpack and it shows you the ID and version ID, although I am unsure exactly where you are doing this. I am on the FTB website looking at the mods, but when I hover over the modpack name and version no IDs are shown.

Modpack in question is Infinity Evolved 

Also ran into this error at a stage if I could get some light shone on it "Background task error for create FTB: no matching manifest for linux/arm/v7 in the manifest list entries" trying to setup using Java8-multiarch

**Comment by itzg** [comment]
This is quite annoying -- they seem to change their webpage layout periodically. Now you have to look at the magic number in the URL:

![image](https://user-images.githubusercontent.com/988985/194679258-20304cf3-6dc0-4909-8ac4-9dfe0074ce4e.png)

So using `FTB_MODPACK_ID=23` worked for me.

I don't seem to have ARM v7 included in the image build, which may have not been supported by the base image. Are you trying to run on a RaspberryPi?

**Reply by TheDoibs** [reply]
Running it on a qnap container

**Reply by itzg** [reply]
If you re-pull, ARMv7 is now included in `itzg/minecraft-server:java8-multiarch`

**Reply by TheDoibs** [reply]
Cheers, I will give it a go

**Reply by TheDoibs** [reply]
Downloaded fine, got this issue when starting the container. 

/start-deployFTBA: line 52: /data/ftb-installer: cannot execute binary file: Exec format error

**Reply by TheDoibs** [reply]
issue with the mod or my settings?

**Reply by TheDoibs** [reply]
![image](https://user-images.githubusercontent.com/104987595/194684641-50eaf646-169d-4fcf-981a-ed3d2b696321.png)

This is my environment settings

**Reply by itzg** [reply]
Ah, it's because FTB only provides arm64 (and x86) downloads of their installer.

**Reply by TheDoibs** [reply]
could I bypass this by installing the mod on a pc and copying the file to the destination drive?

**Reply by itzg** [reply]
Good idea, yes. Then just set `TYPE=FORGE` and `VERSION=1.7.10`.

**Reply by TheDoibs** [reply]
right, I will have a look at the readme for forge options and give it a shot
